### DISCUSSION TITLE
Too much incomplete data in last two days

### DISCUSSION THREAD

**Discussion by djpalmis** [teslamate-org/teslamate#3745] [Q&A]
I just ask for your opinion.
After a little trip in the mountains, when at home, I found out that the trip was completely corrupted by incomplete data.
I closed the drives manually but I think I lost my data.
I use teslamate since years,never happened something like this.
Any idea on what's happening? 
I use teslamate on google cloud, maybe they made some changes? 

Can I close incomplete data manually while car is not in deep sleep?

**Comment by cwanja** [comment]
If your car loses connectivity, it's certainly possible that it's not sending data back to Tesla and thus TeslaMate is incomplete. 

Even historical routes that have worked perfectly, possibly degrade due to telecom signal changes that are beyond Tesla (and TeslaMate) control. 

Yes you can close drives at any point.
